src/codeu/chat/util/TokenizerTest.java

```diff
@@ -0,0 +1,25 @@
+package codeu.chat.util;
+
+import java.io.IOException;
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public final class TokenizerTest {
+  @Test
+  public void testWithQuotes() throws IOException {
+    final Tokenizer tokenizer = new Tokenizer(“hello world how are you”);
+    assertEquals(tokenizer.next(), “hello”);
+    assertEquals(tokenizer.next(), “world”);
+    assertEquals(tokenizer.next(), “how”);
+    assertEquals(tokenizer.next(), “are”);
+    assertEquals(tokenizer.next(), “you”);
```

Comment:
Regular quotation marks (`"..."`) were inexplicably converted to typographic (`“…”`) quotation marks, which are a syntax error. Weird how they got there in the first place.